src/_sass/pages/_how-to-go-contactless.scss

```diff
@@ -0,0 +1,130 @@
+.go-contactless--how-to {
+  .toc {
+    .toc-container {
+      ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+      }
+      a {
+        text-decoration: none !important;
+        border: none;
+        font-weight: 500;
+      }
+    }
+  }
+  .content-container {
+    h2:first-child {
+      margin-top: calc(8rem / 16);
+    }
+    h2 {
+      margin-top: calc(56rem / 16);
+      margin-bottom: calc(48rem / 16);
+    }
+    h3 {
+      margin-top: calc(56rem / 16);
+      margin-bottom: calc(24rem / 16);
+    }
+  }
+}
+
+@include media-breakpoint-up(md) {
+  .how-to-go-contactless .main-primary {
+    height: calc(100vh - 112px);
+    overflow: scroll;
```

Comment:
I found a way to get rid of the extra scrollbars that @thekaveman and I are seeing.

Change this line to
```css
   overflow-y: scroll;
```

and add to `_sass/_bass.scss`:
```css
html {
  overflow-y: unset;
}
```

![image](https://user-images.githubusercontent.com/25497886/208986289-514a0cd4-7e12-4b9e-a868-79e030fd5cf8.png)

I tested the other pages to see if there are any regressions. They seem to behave the same, but please double-check 🙏

Reply:
@machikoyasuda Can you test these changes out on your Mac?

Reply:
(I feel like these changes might not necessarily address the actual question of why the scrollbars show up, but they can at least serve as a temporary solution.)

Reply:
Nice job @angela-tran!! I confirmed both these changes in the deploy preview by editing HTML in devtools:

## Vertical

**Before**

<img width="1268" alt="image" src="https://user-images.githubusercontent.com/1783439/209008304-a980b01c-89f8-4dec-910c-98bbd9824b54.png">

**After**

<img width="1299" alt="image" src="https://user-images.githubusercontent.com/1783439/209008415-e09e358b-b1a6-44e1-a8c8-93460603d3bc.png">

## Horizontal

**Before**

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/1783439/209008841-8d60ad55-b74c-4aea-9a99-e6c7198a6832.png">

**After**

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/1783439/209008895-ce74d907-9a81-4bb5-b031-c6b0231c526a.png">

Reply:
Also @machikoyasuda and I talked about in the new year, fully ripping the CA State Template out of this site, and going with straight Bootstrap. As we know from Benefits, State Template is kind of just getting in the way and weirdly implements some of Bootstrap but not all...

Reply:
Woohoo! Will do.

Reply:
Now available to test: https://deploy-preview-387--cal-itp-mobility-marketplace.netlify.app/how-to/go-contactless

Reply:
Confirmed with @thekaveman  -  Both Windows vertical and horizontal scroll issue resolved.